csharp/FigiApiCsharpExample/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FigiApiCsharpExample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var client = new RestClient("https://api.openfigi.com/v1/mapping");
+            var request = new RestRequest(Method.POST);
+            request.RequestFormat = DataFormat.Json;
+            request.AddHeader("X-OPENFIGI-APIKEY", "");
+            request.AddHeader("Content-Type", "text/json");
+            var list = new List<BloombergRequest>()
+            {
+                new BloombergRequest("US4592001014"),
+                new BloombergRequest("MSFT US Equity"),
+                new BloombergRequest("BBG000BLNNH6")
+            };
+
+            request.AddJsonBody(list);
+
+            var response = client.Post<List<BloombergArrayResponse>>(request);
```

Comment:
How does this behave in the the `error` case?  For instance, it's possible to receive a response that looks like:

```
[{
    "data": [{
        "figi": "..."
        ...
    }]
}, {
    "error": "..."
}]
```

Does the parsing still work?  Is is possible to make, e.g. `OpenFigiResponse` that can parse to a `FigiListResponse` or and `ErrorResponse`?